Source/GUI/GUICommon.h

```diff
@@ -13,6 +13,7 @@ extern QStringList g_memBaseNames;
 
 QString getStringFromType(Common::MemType type, size_t length = 0);
 QString getNameFromBase(Common::MemBase base);
+bool isContainerType(Common::MemType type);
```

Comment:
Naming, do you intend Container vs Struct? Are there plans to expand using Containers as a base for features like bitfields? https://github.com/aldelaro5/dolphin-memory-engine/issues/179

Reply:
Sounds reasonable to me. Thanks for confirming.